### DISCUSSION TITLE
Make red-green vs red-blue selector part of the interactive scorecard

### DISCUSSION THREAD

**Discussion by AmandaBack-NOAA** [NOAA-GSL/MATS#839] [Scorecard - user discussions]
I noticed on the scorecard generation page I have a choice of red-green or red-blue to color code the output. I think it would be nice to add this as one of the "interactive" features on the scorecard, so a card I generate can be easily used by someone with different vision.

**Comment by randytpierce** [comment]
Absolutely, I agree and we will have a color selector so that the colors can be completely customized.
